tests/systemtests/snapshots_test.go

```diff
@@ -0,0 +1,121 @@
+//go:build system_test
```

Comment:
As the commands are really different between simapp and simapp v2, shall we not split the test in two tests?
The multiple if conditions will make it harder to maintain.

We can then just skip the tests when we run with COSMOS_BUILD_OPTIONS v2 and vice versa

Wdyt? cc @tac0turtle

Reply:
should we update the prefix, most diff is that v2 has `store` prefix and v1 has `snapshots`?